pkg/vmimage/data/script.sh

```diff
@@ -11,8 +11,15 @@ xfs_growfs /dev/sda2
 yum -y update -x WALinuxAgent # updating WALinuxAgent kills this script
 yum -y install git libguestfs-tools-c libvirt-daemon-config-network virt-install
 
+CHECKSUM=14ec881815eb9e6618f95df5eb385d961283efc196d97912595ba6484a56180d
 # install golang-1.11 from source because rhel7 ships with golang 1.9 by default
-curl -s https://storage.googleapis.com/golang/go1.11.12.linux-amd64.tar.gz | tar -C /usr/local -xz
+curl -s https://storage.googleapis.com/golang/go1.11.12.linux-amd64.tar.gz --output /tmp/go.1.11.12.linux.amd64.tar.gz
```

Comment:
File here got renamed and then doesn't match the path below :)

```suggestion
curl -s https://storage.googleapis.com/golang/go1.11.12.linux-amd64.tar.gz --output /tmp/go.1.11.12.linux-amd64.tar.gz
```

Reply:
I hate when that happens 😞